DEVELOPMENT.md

```diff
@@ -6,47 +6,69 @@ Because Python is notorious for being difficult to maintain a clean installation
 1. `virtualenv CoilSnake_virtualenv --no-site-packages`
 1. `source CoilSnake_virtualenv/bin/activate`
-  * The above command sets up your CoilSnake virtual development environment. When you open a new terminal for CoilSnake development, always re-run the above command in order to re-enter the virtual development environment. For more information about how this works, see [virtualenv's documentation](https://pypi.python.org/pypi/virtualenv/1.7).
+    1. The above command sets up your CoilSnake virtual development environment. When you open a new terminal for CoilSnake development, always re-run the above command in order to re-enter the virtual development environment. For more information about how this works, see [virtualenv's documentation](https://pypi.python.org/pypi/virtualenv/1.7).
 1. `git clone --recursive https://github.com/mrtenda/CoilSnake.git`
 1. `cd CoilSnake`
 1. `make`
-1. `python setup.py develop`
+1. `python3 setup.py develop`
 
 CoilSnake is now installed in development mode in its own virtualenv, so it does not interfere with other parts of your system. After making code changes to the source, run your code by launching CoilSnake's GUI or CLI:
 
-    python script/gui.py
-    # or...
-    python script/cli.py
+```
+python3 script/gui.py
+# or...
+python3 script/cli.py
+```
 
 ### Using a virtual machine
 
-1. `git clone --recursive https://github.com/mrtenda/CoilSnake.git`
-1. `cd CoilSnake`
-1. `vagrant up ubuntu`
-1. `vagrant ssh ubuntu`
-1. `cd /vagrant`
-1. `sudo python setup.py develop`
+```
+git clone --recursive https://github.com/mrtenda/CoilSnake.git
+cd CoilSnake
+vagrant up ubuntu
+vagrant ssh ubuntu
+cd /vagrant
+sudo python3 setup.py develop
+```
 
 CoilSnake is now installed in development mode on the virtual machine. After making code changes to the source, you can test it by running CoilSnake's CLI interface.
 
-    python script/cli.py
-    
+`python3 script/cli.py`
+
 Please note that the included Vagrant configuration does not run a GUI, meaning that you won't be able to test CoilSnake's GUI with it.
 
+### Creating a standalone Linux executable
+
+1. Follow the Linux steps above and install the additional dependency:
+    1. [pyinstaller](https://www.pyinstaller.org) - `pip install pyinstaller`
+1. Create the CoilSnake executable:
+    1. `python3 setup_exe.py`
+
 ## Windows
 
 1. Install:
-    1. [Python 3.6](https://www.python.org/downloads/release/python-364/) (64-bit version)
-    1. [Visual C++ 2017 Build Tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2017)
-    1. [PyYAML](http://pyyaml.org/wiki/PyYAML)
-        1. `pip install PyYAML-xxx-win_amd64.whl`
+    1. [Python 3.7](https://www.python.org/downloads/release/python-374/) (32-bit version)
+    1. [Visual C++ 2019 Build Tools](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16)
+    1. Select "C++ build tools" under the "Workloads" tab and make sure these are ticked:
```

Comment:
Where is this "Workloads" tab? Is it part of the Python installer, or the Visual C++ 2019 Build Tools installer?

Reply:
Clarified the instructions and reorganized the file to combine the things that are common to both Windows and Linux.